packages/spark-core/components/_accordions.scss

```diff
@@ -12,46 +12,70 @@
 }
 
 .sprk-c-Accordion__item {
-  background-color: $blue-medium;
   margin-bottom: $space-xs;
 
   &:last-child {
     margin-bottom: 0;
   }
+
+  &:first-child {
+    border-top: $accordion-border;
+  }
 }
 
 .sprk-c-Accordion--navigation .sprk-c-Accordion__item {
   background-color: $white;
   margin: 0;
+
+  &:first-child {
+    border-top: 0;
+  }
 }
 
 .sprk-c-Accordion__summary {
-  color: $white;
+  align-items: center;
+  color: $black;
   display: flex;
-  padding: $space-m;
+  padding: $space-m $space-s;
   text-decoration: none;
 
-  &:hover,
-  &:active {
-    background-color: $blue-dark;
-    color: $white;
+  &:hover {
+    color: $accordion-summary-color;
     text-decoration: none;
   }
 }
 
 .sprk-c-Accordion--navigation .sprk-c-Accordion__summary {
   border: 2px solid $gray-medium;
   border-top: 0;
-  align-items: center;
+  padding: $space-m;
+
+  &:hover,
+  &:active {
+    border-bottom: 2px solid $gray-medium;
+    background-color: $blue-dark;
+    color: $white;
+  }
 }
 
 // When open, add same style as when focused
 .sprk-c-Accordion__summary--open {
+  font-weight: bold;
```

Comment:
`sprk-c-Accordion__summary--open` doesnt need the `font-weight: bold` declaration because the heading is already set to bold. If we want to ensure that the text is bold in the case someone decides not to use our ` <h3>` (shouldn't be the case but just trying to help explain my thoughts here) then the ` font-weight: bold` declaration should be on the `sprk-c-Accordion__heading` class that we created for the heading in the summary.